src/navbar/drawer.styles.tsx

```diff
@@ -55,6 +55,7 @@ export const Container = styled.nav<StyleProps>`
 
     ${(props) => VISIBILITY_STYLE(props.$show)}
     ${(props) => {
+        // eslint-disable-next-line no-constant-binary-expression
         const viewHeight = `${props.$viewHeight}px` || "1vh";
```

Comment:
the logic here should be updated to check that `viewHeight` is truthy before using it